Formula/autorest.rb

```diff
@@ -0,0 +1,27 @@
+class Autorest < Formula
+  desc "Swagger (OpenAPI) Specification code generator"
+  homepage "https://github.com/Azure/autorest"
+  url "https://github.com/Azure/autorest/releases/download/AutoRest-0.16.0/autorest.0.16.0.zip"
+  sha256 "bc909aaf3db2105a5d709c940c4d120c5b71047539c2068d484ced1be47650c3"
+
+  depends_on "mono"
+
+  resource "swagger" do
+    url "https://raw.githubusercontent.com/Azure/autorest/master/Samples/petstore/petstore.json"
+    sha256 "8de4043eff83c71d49f80726154ca3935548bd974d915a6a9b6aa86da8b1c87c"
+  end
+
+  def install
```

Comment:
Any chance of a source build instead of installing a binary?

We've made exceptions in the past or certain large/difficult-to-build packages, but we strongly prefer a source build where possible.